src/frontend/src/pages/WorkPackageDetailPage/WorkPackagePage.tsx

```diff
@@ -23,6 +23,7 @@ const WorkPackagePage: React.FC<WorkPackagePageProps> = ({ wbsNum }) => {
   const [editMode, setEditMode] = useState<boolean>(query.get('edit') === 'true');
   const auth = useAuth();
   const isGuest = auth.user?.role === 'GUEST';
+  const isAdmin = auth.user?.role === 'ADMIN' || auth.user?.role === 'APP_ADMIN';
```

Comment:
Probably should validate auth.user in the isloading if statement then move these two things after